shared/local-debug.native.tsx

```diff
@@ -59,7 +59,7 @@ let config = {
 // Developer settings
 if (__DEV__) {
   config.enableActionLogging = true
-  config.enableStoreLogging = false
+  config.enableStoreLogging = true
```

Comment:
🔥

Reply:
It's annoying that I have to change this file to get logging. Really easy to accidentally check into version control

Reply:
I feel the same way about yellow boxes. Until it's bettered here's a git command. I don't understand its documentation but it seems to work ~50% of the time. `git update-index --assume-unchanged local-debug.native.tsx`